hhub/views.py

```diff
@@ -108,3 +121,24 @@ def search_entries(request):
 
 def entry_platform(request, platform):
     return
+
+#########
+# UTILS #
+#########
+def sort_and_order(entries, order_by, sort_by):
+    try:
+        # order_by a specific field from schema
+        if order_by != 1 and order_by != "":
+            # specify an order --> asc is by default, so we're not dealing with it
+            if sort_by != 1 and sort_by != "" and sort_by.lower() != "asc":
+                return entries.order_by("-" + order_by.lower())  # order_by desc order
+            else:
+                return entries.order_by(order_by.lower())        # by default: sort by asc order
+        else:
+            # TODO: should we notify the user about a malformed query in case of empty order_by param? e.g. /api/all?order_by=&page=2
```

Comment:
No, we shouldn't. Instead, we report which query we applied. E.g. if you look at how pagination is implemented, we always report how many elements and which page are we showing (so it's trivial client side to compare them to the actual request to understand what happened)

Reply:
Should we add something like "if group_by has been specified let's write it out in JSONResponse"?

Reply:
I'm not following you, what is `group_by` now?

Reply:
Same as SQL

I mean, should we specify in JSON response what "group_by" has been specified?
e.g. user asks for /api/all?group_by=slug

in JSONResponse:
```....
.....
"group_by": "slug",
[entries]
```

Reply:
Why is group_by needed?